openadapt/ml_models/base_provider.py

```diff
@@ -0,0 +1,36 @@
+from openadapt import models
+from pydantic import BaseModel
+
+class Modality:
+    TEXT = "TEXT"
+    IMAGE = "IMAGE"
+
+class License:
```

Comment:
How about `Use` instead of `License`? The latter we will eventually want to use to refer to specific licenses, e.g. MIT